hexrd/fitting/calibration/calibrator.py

```diff
@@ -25,7 +24,6 @@ def create_lmfit_params(self, current_params):
         for the lattice parameters. The Laue calibrator creates lmfit
         parameters for crystal parameters.
         """
-        pass
```

Comment:
This pass is necessary, please either do not remove it, or raise a `NotImplementedError`.